jobs/future-effective-filings/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[tool.poetry]
+name = "future-effective-filings"
+version = "0.1.0"
+description = ""
+authors = ["karimjazzar <karim@jazzar.ca>"]
+license = "Apache 2.0"
+readme = "README.md"
+packages = [{include = "future_effective_filings"}]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+flask-babel = "^3.1.0"
+flask-moment = "0.10.0"
+flask-script = "2.0.6"
+Flask = "^2.3.2"
+aniso8601 = "8.1.0"
+asyncio-nats-client = "^0.11.5"
+asyncio-nats-streaming = "0.4.0"
+attrs = "^23.1.0"
+certifi = "2020.12.5"
+chardet = "3.0.4"
+ecdsa = "0.14.1"
+flask-jwt-oidc = "^0.3.0"
+flask-restplus = "0.13.0"
+gunicorn = "^20.1.0"
+idna = "2.10"
+jsonschema = "^4.17.3"
+protobuf = "3.14.0"
+pyasn1 = "0.4.8"
+pyrsistent = "0.17.3"
+python-dateutil = "2.8.1"
+python-dotenv = "^1.0.0"
+python-jose = "3.2.0"
+requests = "^2.31.0"
+rsa = "4.6"
+sentry-sdk = "^1.24.0"
+six = "1.15.0"
+urllib3 = "1.26.11"
+legal_api = { git = "https://github.com/bcgov/lear.git", subdirectory = "legal-api", branch = "dev_legal_name_changes" }
```

Comment:
Should just use the business-model, legal-api is far too big

Reply:
actually, I don't think we need to import the business models at all.  

We don't actually use the models in this job.  It just make a call to the internal filings endpoint.

@JazzarKarim I think you can remove this entry from the toml file.

Reply:
Yes, there are no references to it in the code. Alright, sounds good!

Reply:
even better!!